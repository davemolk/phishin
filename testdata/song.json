{"success":true,"total_entries":1,"total_pages":1,"page":1,"data":{"id":979,"slug":"david-bowie","title":"David Bowie","alias":null,"original":true,"artist":null,"lyrics":"David Bowie, David Bowie\nDavid Bowie, David Bowie\nDavid Bowie, David Bowie\nDavid Bowie, David Bowie\n\nUB40, UB40\nUB40, UB40\nUB40, UB40\nUB40, UB40","tracks_count":447,"updated_at":"2021-05-04T13:35:43Z","tracks":[{"id":115,"show_id":11,"show_date":"1986-10-31","venue_name":"Sculpture Room, Goddard College","venue_location":"Plainfield, VT","title":"David Bowie","position":15,"duration":619807,"jam_starts_at_second":null,"set":"2","set_name":"Set 2","likes_count":11,"slug":"david-bowie","tags":[{"id":1,"name":"SBD","priority":1,"group":"Audio","color":"#888888","notes":null,"transcript":null,"starts_at_second":null,"ends_at_second":null},{"id":4,"name":"Jamcharts","priority":4,"group":"Curated Selections","color":"#888888","notes":"Earliest known live version. Jam is played at a slowed tempo initially, but picks up speed and intensity as it develops.","transcript":null,"starts_at_second":null,"ends_at_second":null}],"mp3":"https://phish.in/audio/000/000/115/115.mp3","waveform_image":"https://phish.in/audio/000/000/115/waveform-115.png","song_ids":[979],"updated_at":"2023-10-27T22:29:15Z"}]}}