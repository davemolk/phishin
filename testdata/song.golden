Title:       ID:  Original Artist:  TracksCount:
David Bowie  979  Phish             447

Tracks
ID:  Date:       Venue:                           Location:       Duration:  Mp3
115  1986-10-31  Sculpture Room, Goddard College  Plainfield, VT  10m 19s    https://phish.in/audio/000/000/115/115.mp3
